src/services/cookie.service.js

```diff
@@ -0,0 +1,64 @@
+import Cookies from "universal-cookie";
+
+const cookies = new Cookies();
+const cookieMaxAge = 365 * 86400;
+const cookieName = "macovidvaccines";
+
+let ourCookie = {};
+
+export function getCookie(key) {
+    const theCookie = cookies.get(cookieName, { doNotParse: true });
+    if (theCookie) {
```

Comment:
could you come up with more descriptive names than `theCookie` and `ourCookie`?

Reply:
Good idea! Changing to `encryptedCookie` and `plaintextCookie`.  These aren't exported, so the most important part about them is their encryption state.

Reply:
done